domain/src/main/java/edu/tamu/catalog/domain/model/HoldingsRecord.java

```diff
@@ -43,6 +43,8 @@
 
     private String callNumber;
 
+    private String holdingLocation;
```

Comment:
We should probably refer to it as either `holdingsRecordLocation` or `holdingsLocation`. Pretty sure holdings is the correct lingo.

Reply:
This is the location of a single holding though. Seems weird to pluralize.